From f2e34788c664ec6de316b1c751dc550270be9c96 Mon Sep 17 00:00:00 2001
From: Shinichi Yamashita <yamashitasni@oss.nttdata.co.jp>
Date: Tue, 16 Feb 2016 15:11:57 -0800
Subject: [PATCH 0776/1164] CDH-44190 HIVE-13058 : Add session and
 operation_log directory deletion messages
 (Shinichi Yamashita via Ashutosh Chauhan)

(cherry picked from commit 1b99efec61aaf456f58a584bc56448d32fd65ed8)

Change-Id: Iebf64a601d99f793452f4aa838d36c474d4cfb06
---
 .../hadoop/hive/ql/session/SessionState.java       |    2 ++
 .../hive/service/cli/session/HiveSessionImpl.java  |    2 ++
 2 files changed, 4 insertions(+)

diff --git a/ql/src/java/org/apache/hadoop/hive/ql/session/SessionState.java b/ql/src/java/org/apache/hadoop/hive/ql/session/SessionState.java
index ef799b7..71f670e9 100644
--- a/ql/src/java/org/apache/hadoop/hive/ql/session/SessionState.java
+++ b/ql/src/java/org/apache/hadoop/hive/ql/session/SessionState.java
@@ -709,9 +709,11 @@ public Path getTempTableSpace() {
   private void dropSessionPaths(Configuration conf) throws IOException {
     if (hdfsSessionPath != null) {
       hdfsSessionPath.getFileSystem(conf).delete(hdfsSessionPath, true);
+      LOG.info("Deleted HDFS directory: " + hdfsSessionPath);
     }
     if (localSessionPath != null) {
       FileSystem.getLocal(conf).delete(localSessionPath, true);
+      LOG.info("Deleted local directory: " + localSessionPath);
     }
     deleteTmpOutputFile();
     deleteTmpErrOutputFile();
diff --git a/service/src/java/org/apache/hive/service/cli/session/HiveSessionImpl.java b/service/src/java/org/apache/hive/service/cli/session/HiveSessionImpl.java
index 6abd431..3126eb3 100644
--- a/service/src/java/org/apache/hive/service/cli/session/HiveSessionImpl.java
+++ b/service/src/java/org/apache/hive/service/cli/session/HiveSessionImpl.java
@@ -652,6 +652,8 @@ private void cleanupSessionLogDir() {
     if (isOperationLogEnabled) {
       try {
         FileUtils.forceDelete(sessionLogDir);
+        LOG.info("Operation log session directory is deleted: "
+            + sessionLogDir.getAbsolutePath());
       } catch (Exception e) {
         LOG.error("Failed to cleanup session log dir: " + sessionHandle, e);
       }
-- 
1.7.9.5

